src/sentry/services/hybrid_cloud/organization.py

```diff
@@ -12,36 +25,109 @@
 from sentry.silo import SiloMode
 
 
+@dataclass
+class ApiTeam:
+    id: int = -1
+    status: int = TeamStatus.VISIBLE
+    organization_id: int = -1
+    slug: str = ""
+
+
+@dataclass
+class ApiTeamMember:
+    id: int = -1
+    is_active: bool = False
+    role: Optional[TeamRole] = None
+    project_ids: List[int] = field(default_factory=list)
+    scopes: List[str] = field(default_factory=list)
+    team_id: int = -1
+
+
+@dataclass
+class ApiProject:
+    id: int = -1
+    slug: str = ""
+    name: str = ""
+    organization_id: int = -1
+    status: int = ProjectStatus.VISIBLE
+
+
 @dataclass
 class ApiOrganizationMember:
```

Comment:
I know we plan to duplicate organization members on the control silo, so this will be one of the first incarnations of that modus operandi

Reply:
Yes, although there is a critical open question: if organization members exist in both silos, when I need to create an organization member from the pipeline flow... who receives the call?  which silo's representation is "most" authoritative?    Also because we need both the organization and organization member and team memberships to all agree (have consistency), we have to pull them all from the same source, we can't split that across two places.

It seems like creating new organization members will *first* call control silo to save since it is associated strongly with login (and we require read after write properties in that case).  So then... how does the "region" organization member get created?  @markstory 

This needs to be solved before auth can be finalized.

Reply:
Ok.  So `ProjectTeam`, `OrganizationMemberTeam` and such also Region silo?  Do those also propagate to control silo async?  Also, if we write to organization member in the region silo first, that means that the organization service *should not* be reading organization members from the control silo, as that would break read after write properties.  @mikejihbe